src/facade/cmd_arg_parser.h

```diff
@@ -143,6 +143,13 @@ struct CmdArgParser {
     return std::exchange(error_, {});
   }
 
+  bool HasAtLeast(size_t i) const {
+    if (i == 0) {
```

Comment:
it looks strange for me, we always should have 0 elements

Reply:
+1